thirdparty/icu4c/common/uchar.cpp

```diff
@@ -698,6 +698,7 @@ uchar_addPropertyStarts(const USetAdder *sa, UErrorCode *pErrorCode) {
     sa->add(sa->set, u'A');
     sa->add(sa->set, u'Z'+1);
     // fullwidth
+#if 0
```

Comment:
Unicode char/string literals are valid since C++11, you are probably missing `/utf-8` flag for MSVC if this is causing problems.